### DISCUSSION TITLE
Error in R Studio server SLURM output

### DISCUSSION THREAD

**Discussion by yaaminiv** [RobertsLab/resources#1222] [Q&A]
## TL;DR Is there something in my R Studio sever SLURM script that could be causing these errors?

I'm getting the following error in the SLURM output file when launching R Studio server:

![Screen Shot 2021-05-24 at 9 39 57 AM](https://user-images.githubusercontent.com/22335838/119379534-73cd6180-bc74-11eb-907d-7ecf2113ee9b.png)

This is the code I'm using to launch the R Studio server session: https://github.com/RobertsLab/project-oyster-oa/blob/master/code/Haws/04-DSS-RStudio.sh

I encountered some difficulty saving my R code earlier, but forgot to screenshot the error from R Studio. I ended up having to terminate R. This was the error that I got after R was terminated and the R Studio sever loaded (note: I did not kill the SLURM script or log out):

![Screen Shot 2021-05-24 at 9 47 31 AM](https://user-images.githubusercontent.com/22335838/119380040-0ff76880-bc75-11eb-95a0-06558cb1a589.png)

**Comment by yaaminiv** [comment]
![Screen Shot 2021-05-24 at 2 03 05 PM](https://user-images.githubusercontent.com/22335838/119407203-c456b600-bc98-11eb-86fd-ea465952e80f.png)

Error might be originating from the fact that I may have exceeded my disk quota? I tried to move my singularity container from my user directory to the `/gscratch/srlab/programs/singularity_containers/` but I don't have write access. @kubu4 can I get write access to do this?

**Reply by kubu4** [reply]
Use your `/gscratch/srlab/$USER/` directory. Crate a Singularity subdirectory, if you'd like.

**Reply by yaaminiv** [reply]
Moving the container didn't stop the error from being produced:

![Screen Shot 2021-05-24 at 3 38 45 PM](https://user-images.githubusercontent.com/22335838/119415317-37b2f480-bca6-11eb-8807-fbb70edceab8.png)

This is the error that I get when opening R Studio server:

![Screen Shot 2021-05-24 at 3 40 45 PM](https://user-images.githubusercontent.com/22335838/119415409-69c45680-bca6-11eb-8d24-7d24e4429493.png)

Is there something originating from my SLURM script?
